chart/templates/metadata.yaml

```diff
@@ -19,11 +19,12 @@ spec:
     contactGivenName: GOV.UK Verify
     contactSurname: Support
     contactEmail: idasupport@digital.cabinet-office.gov.uk
+    validityDays: 14
   enabled: true
   samlSigningCertRequest:
     countryCode: GB
     commonName: Verify Proxy Node SAML Signing
-    expiryMonths: 2
+    expiryMonths: 3
```

Comment:
Shouldn't this be 9 months? The plan is they expire after 9 months but we rotate after 3, right?

Reply:
I guess but if we plan to rotate them every 3 months then it's effectively the same. We can change this to 9 months if needed when we implement the cert rotation in VMC